packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.css

```
@@ -166,3 +166,9 @@
 .relatedTitle {
     composes: sectionTitle;
 }
+
+.quantityRoot {
+    composes: root from '../CartPage/ProductListing/quantity.css';
+    grid-template-columns: auto 1fr auto;
+    padding: 0 1rem;
```

@huykon - I would suggest this change given @schensley's feedback on sizing. This is no longer a dropdown, so I don't think anything else needs changed other than the width.
```suggestion
    grid-template-columns: auto 4rem auto;
    justify-content: start;
```

---

I have just updated code as your recommend.